Core/HLE/sceKernel.cpp

```diff
@@ -261,6 +263,7 @@ void __KernelDoState(PointerWrap &p)
 		__UmdDoState(p);
 		__UtilityDoState(p);
 		__UsbDoState(p);
+		__UsbGpsDoState(p);
```

Comment:
New DoState functions should be added at the end of the list, for compatibility with old states. Actually come to think of it, that might not be an issue after all, but that's how we've always done it so please do so anyway in case I'm missing something...

Reply:
done